calyx-py/calyx/py_ast.py

```diff
@@ -112,7 +204,7 @@ class CombComponent:
     """Like a Component, but with no latency and no control."""
 
     name: str
-    attributes: list[Attribute]
+    attributes: set[Attribute]
```

Comment:
not for this PR: but it may make sense in the long run to separate the set
attributes from the normal attributes. For now this is fine since we only have
`@pos` as a set attribute

Reply:
Will add this to my TODOs!